lib/src/center_of_mass.dart

```diff
@@ -0,0 +1,61 @@
+//import 'helpers.dart';
+//import 'invariant.dart';
+import 'package:turf/meta.dart';
+import 'centroid.dart';
+
+Feature<Point> centerOfMass(
+  GeoJSONObject geoJson,
+  {Map<String, dynamic>? properties = const {}, }) 
+
+  {
+  List<List<double>> coords =[];
```

Comment:
`List<Position> coords` -> we have this `Position` type instead of `List<double>` in turf dart for better typisation